src/tests/Regressions/coreclr/0416/hello.cs

```diff
@@ -1,15 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 using System;
+using Xunit;
 
 public class Hello
 {
-	public static int Main()
-	{
-		Console.WriteLine("Make sure to set the following env vars:");
-		Console.WriteLine("  DOTNET_LogEnable=1");
-		Console.WriteLine("  DOTNET_LogToConsole=1");
-		Console.WriteLine("This test ensures that logging is working");
-		return 100;
-	}
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        Console.WriteLine("Make sure to set the following env vars:");
+        Console.WriteLine("  DOTNET_LogEnable=1");
+        Console.WriteLine("  DOTNET_LogToConsole=1");
+        Console.WriteLine("This test ensures that logging is working");
+    }
```

Comment:
This test doesn't seem to be doing that.

This isn't related to your change except that I was going to comment that the comment for RPI should indicate the environment variables, except...

(I wouldn't hold this change up - just file an item or even just delete this test)

Reply:
Thanks Mark for spotting, I do admit I haven't scrutinized every single test but this one does look weird. I'm going to push a commit deleting it and, if someone complains, we can bring it back and possibly try to make sure it does something reasonable.